hellonode/server.js

```diff
@@ -14,12 +14,23 @@
  * limitations under the License.
  */
 
-var http = require('http');
+const http = require('http');
+const os = require('os');
+
+const port = process.env.PORT || 8080;
```

Comment:
nit: do you want to make this more unique than PORT? possible to already exist on system.